src/_data/side-nav.yml

```diff
@@ -97,10 +97,12 @@
       permalink: /development/platform-integration/platform-channels
   - title: Packages & Plugins
     children:
-    - title: Using packages
+    - title: "Using packages"
```

Comment:
You don't need to add quotes around the title here.

Reply:
I know. But it doesn't hurt, either.   ;)